nexus/users/tests/test_models.py

```diff
@@ -10,7 +10,7 @@ class UserModelTestCase(TestCase):
     def test_create_user(self):
         u = User.objects.create_user(
             email='f@f.com', password='abc', first_name="F", last_name='B',
-            gender='others', tshirt_size='large', phone_number='+910123456789'
+            gender='others', tshirt_size='large', phone_number='+919876543210'
```

Comment:
Why this phone number change?

Reply:
Is it because of validations of PhoneNumberField?

Reply:
Yes. The previous versions ignored if the number was not valid. This change was introduced in a 3.0.1. 
https://github.com/stefanfoulis/django-phonenumber-field/commit/b3ff5c387ace44203cf646c6513aadcd893fcc38